openapi.yaml

```diff
@@ -6356,6 +6356,53 @@ paths:
       - lang: CLI
         source: >
           linode-cli linodes upgrade 123
+  /linode/instances/{linodeId}/nodebalancers:
+    parameters:
+    - name: linodeId
+      in: path
+      description: ID of the Linode to look up
+      required: true
+      schema:
+        type: integer
+    x-linode-cli-command: linodes
+    get:
+      x-linode-grant: read_only
+      tags:
+      - Linode Instances
+      summary: Linode NodeBalancers View
+      description: Returns a list of NodeBalancers that are assigned to this Linode.
+      operationId: getLinodeNodeBalancers
+      x-linode-cli-action: nodebalancers
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - linodes:read_only
```

Comment:
Tokens must have at least read_only access to the linode requested and to the nodebalancers assigned. Does this get noted here?

Reply:
Yes that should get noted, though I think it's more accurate to note NodeBalancer permission requirements in the endpoint description. Submitting a revision to that effect...